apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/model/SimpleXAPI.kt

```diff
@@ -1018,12 +1020,13 @@ object ChatController {
     }
   }
 
-  suspend fun apiUpdateChatItem(rh: Long?, type: ChatType, id: Long, itemId: Long, mc: MsgContent, live: Boolean = false): AChatItem? {
-    val r = sendCmd(rh, CC.ApiUpdateChatItem(type, id, itemId, mc, live))
+  suspend fun apiUpdateChatItem(rh: Long?, type: ChatType, id: Long, itemId: Long, um: UpdatedMessage, live: Boolean = false): AChatItem? {
```

Comment:
what are you trying to save by writting um versus updatedMessage? Characters? No need, just use names of variables where you understand the meaning without doing mind gymnastics.
um: UpdatedMessage -> updatedMessage: UpdatedMessage

Reply:
no need

Reply:
It's debatable whether you want names to repeat the meaning already conveyed in types.

It's less cognitive load to have short names and descriptive types, especially with short functions.

Reply:
type name doesn't represent variable meanting. You have hundreds of ChatItem but you don't know what's inside if you don't call it `chatItem: ChatItem, updatedChatItem: ChatItem`

Reply:
and when you read the code, you will only see `um = something` from the calling side without knowing what this `um` means because you will not see type name

Reply:
shorteing variable name is useful inside a local function when you need to call it often but not inside a parameter name. All globals should have clear name always